pkgs/development/compilers/llvm/11/llvm/default.nix

```diff
@@ -15,6 +15,8 @@
 , zlib
 , buildLlvmTools
 , debugVersion ? false
+, doCheck ? stdenv.isLinux && (!stdenv.isx86_32) && (!stdenv.hostPlatform.isMusl) && (!stdenv.hostPlatform.isRiscV)
+  && (stdenv.hostPlatform == stdenv.buildPlatform)
```

Comment:
Just in case anyone else was also wondering: #211728 copies this condition (which LLVM 9 and 11 have) and disables the tests on RISC-V hosts for LLVM 12 - 14 (though not LLVM 10...); it'll have to be adjusted once this PR is merged.